datatables.go

```diff
@@ -65,6 +66,81 @@ func (p *Parser) bindEntities() {
 	p.bindBombSites()
 	p.bindPlayers()
 	p.bindWeapons()
+	p.bindBomb()
+}
+
+func (p *Parser) bindBomb() {
+	bombState := p.gameState.bombState
+
+	// Track bomb when it's not held by a player
+	p.stParser.ServerClasses().FindByName("CC4").OnEntityCreated(func(bomb *st.Entity) {
+		bombState.Position = bomb.Position()
```

Comment:
Shouldn't be neccessary, the `OnPositionUpdate` handler should be called with the initial value.